### PR TITLE
dictlib-feedstock

### PR DESCRIPTION
**Pull request by Tata17**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vend other packages
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/elephas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - cython
+    - flask
+    - keras
+    - pip
+    - pyspark
```
Comment (scopatz) [inline recipes/elephas/meta.yaml]: Are pyspark, keras, and flask needed as host deps?

File: recipes/elephas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - python
+    - six
+
+  run:
+    - cython
```
Comment (scopatz) [inline recipes/elephas/meta.yaml]: Is cython a runtime dep?

File: recipes/elephas/meta.yaml (hunk 1 above)
Comment (Tata17) [reply recipes/elephas/meta.yaml]: @scopatz I just realized that I didn't take elephas out while working on dictlib and that is the recipe you are looking at (elephas) I will take out elephas (for which I am troubleshooting the Windows build) and refresh the PR for dictlib. I am sorry about that.

**Comment by Tata17** [comment]
@conda-forge/help-python, this recipe is ready for review

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**